app/views/lookup_keys/_fields.html.erb

```diff
@@ -4,9 +4,9 @@
   <%= remove_child_link(_("Remove %s?") % (f.object.new_record? ? _("Variable") : f.object), f, { :class => 'btn btn-danger hide' }) if controller_name ==  "puppetclasses" && !is_param %>
   <fieldset>
     <h2><%= _("Parameter details") %></h2>
-    <%= text_f f, :key, :disabled => is_param, :size => "col-md-8" %>
+    <%= text_f f, :key, :disabled => is_param, :size => "col-md-8", :label => _('Key')  %>
```

Comment:
Note sure that model attributes need extracting, they're already in model_attributes.rb (but probably under the wrong name as 6b4b489 seemed to rename them).

This will mean twice the number of translations, but only one gets used. Better fix this separately if it's really an issue, it's unrelated.